libraries/AP_NavEKF2/AP_NavEKF2_core.cpp

```diff
@@ -758,6 +757,24 @@ void NavEKF2_core::calcOutputStates()
     // sum delta velocities to get velocity
     outputDataNew.velocity += delVelNav;
 
+    // Implement third order complementary filter for height and height rate
+    // Reference Paper :
+    // Optimizing the Gains of the Baro-Inertial Vertical Channel
+    // Widnall W.S, Sinha P.K,
+    // AIAA Journal of Guidance and Control, 78-1307R
+
+    // Perform filter calculation using backwards Euler integration
+    // Coefficients selected to place all three filter poles at omega
+    const float CompFiltOmega = 10.0f;
```

Comment:
This can be increased to use baro across wider bandwidth which reduces reliance on IMU and vice-versa

Reply:
The filter integrates IMU to get vertical velocity and position and then corrects using baro, so the bigger this gain, the more the baro corrects.

Reply:
I tried 3 different values for CompFiltOmega (10, 30 and 50) and in SITL at least I didn't really notice much difference in performance.  I have logs if that's of interest.